src/js/pages/catalog/PackagesTab.js

```diff
@@ -38,6 +39,30 @@ const PackagesBreadcrumbs = () => {
   return <Page.Header.Breadcrumbs iconID="catalog" breadcrumbs={crumbs} />;
 };
 
+const PackagesEmptyState = () => {
+  return (
+    <AlertPanel>
+      <AlertPanelHeader>No package repositories</AlertPanelHeader>
+      <p className="tall">
+        You need at least one package repository with some packages to be
+        able to install packages. For more {" "}
+        <Link
+          target="_blank"
+          to={MetadataStore.buildDocsURI("/administering-clusters/repo")}
```

Comment:
Uh oh...this generates: `href="#https://docs.mesosphere.com/1.11/administering-clusters/repo"`

I think just use bare `<a>` tag.